frame/example-offchain-worker/src/lib.rs

```diff
@@ -40,20 +40,35 @@
 //! one unsigned transaction floating in the network.
 #![cfg_attr(not(feature = "std"), no_std)]
 
+use frame_system::{
+	self as system,
+	ensure_signed,
+	ensure_none,
+	offchain::{
+		AppCrypto,
+		CreateSignedTransaction,
+		SendUnsignedTransaction,
+		SendRawUnsignedTransaction,
+		SignedPayload,
+		SigningTypes,
+		Signer,
+	}
```

Comment:
```suggestion
	self as system, ensure_signed, ensure_none,
	offchain::{
		AppCrypto, CreateSignedTransaction, SendUnsignedTransaction, SendRawUnsignedTransaction, SignedPayload, SigningTypes, Signer,
	}
```

Reply:
Is there a rule for such a format? i prefer my version TBH as the vertical version is more readable